packages/flutter_plugin_android_lifecycle/example/android/app/build.gradle

```diff
@@ -24,13 +24,13 @@ if (flutterVersionName == null) {
 
 
 android {
-    compileSdk flutter.compileSdkVersion
+    compileSdk = flutter.compileSdkVersion
 
 
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
         applicationId "io.flutter.plugins.flutter_plugin_android_lifecycle_example"
-        minSdkVersion flutter.minSdkVersion
+        minSdkVersion = flutter.minSdkVersion
         targetSdkVersion flutter.targetSdkVersion
```

Comment:
Why `minSdkVersion` but not `targetSdkVersion`?

Is the plan just to wait and fix the assignment type when standardizing on `targetSdk`?

Reply:
accidental add, removing. I do want to move us to use equals and minSdk and targetSdk but only with the enforcement to keep it that way.